<ui version="4.0" >
 <class>ConfigDglLexikaClass</class>
 <widget class="QWidget" name="ConfigDglLexikaClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="leftMargin" >
    <number>4</number>
   </property>
   <property name="topMargin" >
    <number>4</number>
   </property>
   <property name="rightMargin" >
    <number>4</number>
   </property>
   <property name="bottomMargin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Wörterbuch 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="language1" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="infoLanguage1" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Wörterbuch 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="language2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="infoLanguage2" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
